projects/Magnetosphere/Magnetosphere.cpp

```diff
@@ -726,7 +726,7 @@ namespace projects {
             // We don't care about cells that are too far from the ionosphere
             const Real beta {P::useJPerB ? std::log2(cell->parameters[CellParams::AMR_JPERB]) + logDx + P::JPerBModifier + refLevel : 0.0};
             bool shouldRefine = cell->parameters[CellParams::AMR_ALPHA] > P::refineThreshold || beta > 0.5;
```

Comment:
:oof: please do not use beta for checking alpha_2! Beta is a fundamental parameter of plasmas, there's a high risk of causing confusion with this notation. Admittedly, plasma beta cannot be negative,but still, in a plasma code let's resevere beta for plasma beta.

Reply:
Renamed `alphaTwo` to match the paper draft